### PR TITLE
M3-1889 tests for auto backup enrollment M3-1789 and M3-1793 

### PR DESCRIPTION
**Pull request by nwisnewski**
**Purpose**
Added tests for the auto enroll in backups for future linode global setting, dashboard backup CTA, linodes backup CTA, and enroll existing linode for backups features.

**To Test**
`yarn && yarn start` new shell `yarn selenium` new shell `yarn e2e:modified`

### CODE REVIEW

File: e2e/specs/globalsettings/auto-backup-enrollment.spec.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        EnableAllBackupsDrawer.drawerClose.click();
+        EnableAllBackupsDrawer.drawerBase.waitForVisible(constants.wait.normal,true);
+    });
+
+    it('Backup auto enrollment CTA should link to globabl settings page', () => {
+        Dashboard.autoBackupEnrollmentCTA.click();
+        GlobalSettings.baseElementsDisplay();
+    });
+
+    it('The gloabal settings page contains an external link to the backups pricing page', () => {
```
Comment (patthiel) [inline e2e/specs/globalsettings/auto-backup-enrollment.spec.js]: spelling 😉

File: e2e/specs/globalsettings/auto-backup-enrollment.spec.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        expect(EnableAllBackupsDrawer.drawerTitle.getText()).toEqual('Enable All Backups');
+    });
+
+    it('Enable all backups drawer contains an external link to the backup pricing page', () => {
+        EnableAllBackupsDrawer.backupPricingPage.click();
+        checkBackupPricingPageLink();
+        EnableAllBackupsDrawer.enableAllBackupsDrawerDisplays(true);
+    });
+
+    it('Enable all backups drawer displays the count of linodes to be backed up in message and linodes to be backedup', () => {
```
Comment (patthiel) [inline e2e/specs/globalsettings/auto-backup-enrollment.spec.js]: missing space at the end of the title

File: e2e/specs/globalsettings/auto-backup-enrollment.spec.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    });
+
+    it('The enable all backups drawer can be opened from the global settings page if auto backups is not enabled and there are linodes not backed up', () => {
+        GlobalSettings.enableBackupsForAllLinodesDrawer.click();
+        EnableAllBackupsDrawer.enableAllBackupsDrawerDisplays(true);
+        EnableAllBackupsDrawer.drawerClose.click();
+        EnableAllBackupsDrawer.drawerBase.waitForVisible(constants.wait.normal,true);
+    });
+
+    it('Enable backup auto enrollment toggel', () => {
```
Comment (patthiel) [inline e2e/specs/globalsettings/auto-backup-enrollment.spec.js]: toggle*

File: e2e/specs/globalsettings/auto-backup-enrollment.spec.js (hunk 1 above)
Comment (nwisnewski) [reply e2e/specs/globalsettings/auto-backup-enrollment.spec.js]: Gloabal not is word?

File: e2e/specs/globalsettings/auto-backup-enrollment.spec.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+        GlobalSettings.enrollInNewLinodesAutoBackupsToggle.click();
+        browser.waitUntil(() => {
+            return GlobalSettings.enrollInNewLinodesAutoBackupsToggle.getAttribute('data-qa-toggle') === 'true';
+        }, constants.wait.normal);
+        expect(retrieveGlobalSettings().backups_enabled).toBe(true);
+    });
+
+    it('Backups should be enabled when creating a new linode and checkbox', () => {
+        GlobalSettings.globalCreate.click();
```
Comment (patthiel) [inline e2e/specs/globalsettings/auto-backup-enrollment.spec.js]: We have a utility method in the base page that does this series of steps  (Lines 102-104)), i think `selectGlobalCreateItem('Linode')` will do it

**Pull request merged**